### PR TITLE
Fix wrong return type (solves #124)

### PR DESCRIPTION
**Pull request by JanKrb**
Solves #124 

I completely removed the return type specification to match the other files.

### CODE REVIEW

**Comment by spekulatius** [comment]
Hey @JanKrb 

thanks for picking this up. I'm considering if it would make sense to add return types wherever possible. This follow closely with Spatie's approach. I think it was in this [video](https://www.youtube.com/watch?v=woW00J0oEqw). What is your take on this approach?

Cheers,
Peter

**Comment by JanKrb** [comment]
Hey,

I totally agree with you. I only removed them because most of them don't have them either. I'll add them just now. :)

**Comment by JanKrb** [comment]
I added the return types for the method shown in #124 and the methods in the same section. There might be a whole lot a other methods without return type.

File: russian.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ public function getFullNameAttribute()
 Хорошо:
 
 ```php
-public function getFullNameAttribute(): bool
+public function getFullNameAttribute()
```
Comment (spekulatius) [inline russian.md:71]: @JanKrb should we add it here too?

File: README.md [hunk 2]
```diff
@@ -110,7 +110,7 @@ public function getFullNameAttribute()
 Good:
 
 ```php
-public function getFullNameAttribute(): string
+public function getFullNameAttribute()
```
Comment (spekulatius) [inline README.md:113]: @JanKrb should we add it here too?

File: russian.md (hunk 1 above)
Comment (JanKrb) [reply russian.md]: Looks like I missed this one. Fixed.

File: README.md (hunk 2 above)
Comment (JanKrb) [reply README.md]: Looks like I missed this one. Fixed.

**Comment by spekulatius** [comment]
Thanks for adding it in. That's already a good step forward :+1:

**Pull request merged**